test/Microsoft.AspNet.PlatformHandler.FunctionalTests/HelloWorldTest.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Server.Testing;
+using Microsoft.AspNet.Testing.xunit;
+using Microsoft.Framework.Logging;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNet.Loader.IIS.FunctionalTests
+{
+    // Uses ports ranging 5061 - 5069.
```

Comment:
I'm sure this had been discussed before. But shall we find the free port in a more dynamic fashion?
https://github.com/aspnet/dnx/blob/dev/test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/Infrastructure/DthTestServer.cs#L135